ntoskrnl/kdbg/kdb_cli.c

```diff
@@ -1051,17 +1051,17 @@ KdbpTrapFrameFromPrevTss(
     return TRUE;
 }
 
-VOID __cdecl KiTrap02(VOID);
+VOID __cdecl  KiTrap02Handler(VOID);
 VOID FASTCALL KiTrap03Handler(IN PKTRAP_FRAME);
-VOID __cdecl KiTrap08(VOID);
-VOID __cdecl KiTrap09(VOID);
+VOID __cdecl  KiTrap08Handler(VOID);
+VOID FASTCALL KiTrap09Handler(IN PKTRAP_FRAME);
 
 static BOOLEAN
 KdbpInNmiOrDoubleFaultHandler(
-    ULONG_PTR Address)
+    PVOID Address)
```

Comment:
Why?

Reply:
Same pattern as #1996, easier to read, ...

Reply:
> easier to read

Totally not.